### DISCUSSION TITLE
Fantastic stuff, I have a question though.

### DISCUSSION THREAD

**Discussion by Kradek200** [anthony1x6000/ROBLOX2016stylus#17] [General]
Where in the script is the part for the Roblox Logo in top left corner located? I'd like to have the 2017 and onwards one and i've been searching for keywords like "Roblox" "logo" "home" Etc but none of them affected the icon. Help would be much appreciated!

**Comment by anthony1x6000** [comment]
If you have 1.2.2 release look at line 3426.
Roblox has a new website policy thing that blocks all unverified domains so you can't use something like imgur to host images. You'll have to use something on the roblox cdn. You can also just upload an image as a decal and use that as an image source. 
```
    .light-theme .icon-default-logo, .light-theme .icon-logo {
        background-image: url('[imagelink]');
        background-size: 118px 30px;
    }
```

**Reply by anthony1x6000** [reply]
The classes that the logo fall under on the site itself are "icon-logo" and "icon-logo-r" if that helps.

**Reply by Kradek200** [reply]
After a bit of fiddling around with it I got what I wanted, thanks alot!

**Comment by sidjadon** [comment]
Roblox [JJSploit](https://gameinstants.com/how-to-jjsploit-download/) can be helpful.
